src/open_inwoner/cms/cases/views/status.py

```diff
@@ -141,6 +141,59 @@ def get_context_data(self, **kwargs):
             self.store_statustype_mapping(self.case.zaaktype.identificatie)
             self.store_resulttype_mapping(self.case.zaaktype.identificatie)
 
+            # TODO: restore
+            # objectcontactmomenten = []
+            # if contactmoment_client := build_client_openklant("contactmomenten"):
+            #     objectcontactmomenten = (
+            #         contactmoment_client.retrieve_objectcontactmomenten_for_zaak(
+            #             self.case
+            #         )
+            #     )
+            # questions = [ocm.contactmoment for ocm in objectcontactmomenten]
+            # from .. import dev_snippets
+            # questions = dev_snippets.contactmomenten_stubs()
+
+            # TODO: remove
+            import datetime
+
+            from deteutil import tz
+            from zgw_consumers.api_models.base import factory
+
+            from open_inwoner.openklant.api_models import ContactMoment
+
+            questions = []
+            for i in range(10):
+                data = {
+                    "url": "https://contactmomenten.nl/api/v1/contactmoment/aaaaaaaa-aaaa-aaaa-aaaa-bbbbbbbbbbb2",
+                    "bronorganisatie": "Zbw",
+                    "registratiedatum": datetime.datetime(
+                        1985 + i,
+                        12,
+                        2,
+                        0,
+                        51,
+                        24,
+                        tzinfo=tz.getzz("Europe / Amsterdam"),
+                    ),
+                    "kanaal": "Contactformulier",
+                    "tekst": "Lorem ipsum dolor sit amet?",
+                    "medewerker_identificatie": None,
+                    "identificatie": "AB123",
+                    "type": "SomeType",
+                    "onderwerp": "e_suite_subject_code",
+                    "status": "afgehandeld",
+                    "antwoord": "yes",
+                    "voorkeurskanaal": "sZCjHWnImiafznWmnrFlHYCpEqIyysxDxOCHnKrslJYkZuADJE",
+                    "voorkeurstaal": "Avo",
+                    "vorig_contactmoment": None,
+                    "volgend_contactmoment": None,
+                    "onderwerp_links": ["https://www.kramer.biz/"],
+                    "initiatiefnemer": "klant",
+                    "medewerker": "https://davis.com/",
+                }
+                questions.append(factory(ContactMoment, data))
+            questions.sort(key=lambda q: q.registratiedatum, reverse=True)
+
```

Comment:
Hideous, I know, but atm it's difficult for Jiro (or anyone else) to actually test the toggle/hide behavior, since our mocks don't provide enough contactmomenten.

Reply:
If this is the apimocks in the repo then we could add more. I think it'd work to just duplicate the items in the list resources (without even changing the uuids etc)

Reply:
The uuids need to be checked as well. I've created a separate ticket, though it's not urgent ([https://taiga.maykinmedia.nl/project/open-inwoner/issue/2196](https://taiga.maykinmedia.nl/project/open-inwoner/issue/2196)).